phpunit/block-editor-test.php

```diff
@@ -239,16 +239,6 @@ function test_get_default_block_editor_settings() {
 		$this->assertInternalType( 'int', $settings['maxUploadFileSize'] );
 	}
 
-	/**
-	 * @ticket 52920
-	 */
-	function test_get_block_editor_settings_returns_default_settings() {
```

Comment:
This test is bad conceptually, at some point the default settings and the block editor settings might diverge. More importantly, in the context of the plugin hooks applied change the output of `gutenberg_get_block_editor_settings` 😄